docs/configuration.rst

```diff
@@ -59,6 +65,44 @@ For example, if you want your custom fragment types to be ``["feat", "fix", "cho
    showcontent = false
 
 
+.. warning::
+
+   Since TOML mappings aren't ordered, the sections are always rendered alphabetically.
```

Comment:
I guess you can have an "order_index" and sort the dict in jinja or python.

But this requires extra configuration work and extra template or python work.

Reply:
Yeah it just doesn't make sense to force an ordering on an unordered data structure.

We should focus on making the ordered version nicer (better key names, make `showcontent` optional).